README.md

```diff
@@ -121,53 +157,23 @@ export GAZEBO_MODEL_PATH=${GAZEBO_MODEL_PATH}:~/catkin_ws/src/avoidance/sim/mode
 export QT_X11_NO_MITSHM=1
 
 # Setup some more Gazebo-related environment variables
-. ~/Firmware/Tools/setup_gazebo.bash ~/Firmware ~/Firmware/build/posix_sitl_default
+. ~/Firmware/Tools/setup_gazebo.bash ~/Firmware ~/Firmware/build/px4_sitl_default
 
 # Build and run simulation
-make posix_sitl_default gazebo
+make px4_sitl_default gazebo
 ```
 
-12. Add the Firmware directory to ROS_PACKAGE_PATH so that ROS can start the PX4.
+4. Add the Firmware directory to ROS_PACKAGE_PATH so that ROS can start the PX4.
 
 ```bash
 export ROS_PACKAGE_PATH=${ROS_PACKAGE_PATH}:~/Firmware
 ```
 
 You should now be ready to run the simulation.
 
-## Installation for Ubuntu 14.04 and ROS Indigo
+### Global Planner
 
-A full step-by-step guide is not available for this configuration. You might find the [instructions for Ubuntu 16.04 and ROS Kinetic](#installation-for-ubuntu-1604-and-ros-kinetic) useful, though. Another source of information is also the [PX4 dev guide](http://dev.px4.io).
-
-Notable differences with ROS Kinetic are:
-
-* Installing the pointcloud library requires another package from another repository:
-
-```bash
-# Install PCL
-sudo add-apt-repository ppa:v-launchpad-jochen-sprickerhof-de/pcl
-sudo apt-get update
-sudo apt-get install libpcl-all
-```
-
-* Octomap has different packages for Indigo:
-
-```bash
-# Install the Octomap Library
-sudo apt-get install ros-indigo-octomap ros-indigo-octomap-mapping
-```
-
-# Running the Planner in Simulation
-
-## Simulating a depth camera
-
-1. Make sure that you have sources the catkin setup.bash from your catkin workspace.
-
-```bash
-source ~/catkin_ws/devel/setup.bash
-```
-
-2. Run the simulation.
+To run the simulation.
 
 ```bash
 roslaunch global_planner global_planner_sitl_mavros.launch
```

Comment:
Below this point you say "To start flying, put the drone in OFFBOARD mode and arm it.". This is info to capture - are we only going to show OFFBOARD mode and arming? Does this demo only work in OFFBOARD mode. Does global planner only work for offboard mode?

I know the answer, I'm suggesting it might be worth capturing what we can and cannot show in this demo.